tests/loader_wsi_tests.cpp

```diff
@@ -445,6 +525,46 @@ TEST_F(WsiTests, GetPhysicalDeviceXlibPresentICDSupport) {
 
     ASSERT_EQ(VK_TRUE, env->vulkan_functions.vkGetPhysicalDeviceXlibPresentationSupportKHR(physical_device, 0, nullptr, 0));
 }
+
+TEST_F(WsiTests, XlibGetPhysicalDeviceSurfaceSupportKHR) {
+    FrameworkEnvironment env{};
```

Comment:
This is wrong. TEST_F(WsiTests, provides a framework environment, so this is going to cause confusion as two test frameworks are vying for control.

Reply:
So my tests failed if I mixed `TEST_F` with `TEST`, so to resolve this, I converted all the WSI tests to `TEST`.